jython/loconet/LnEditEvolutionRoutesAndOpSw.py

```diff
@@ -143,40 +151,42 @@ def __init__(self) :
                                 [47,'Empty'],
                                 [47,'Empty']
                                 ]])
+        self.deviceCVAttributes.append([0x03,'DS64',[],8,[]])
+
     def isDeviceNameValid(self,deviceName) :
         for dev in self.deviceCVAttributes :
             if dev[DeviceAttributes.ColName]==deviceName :
-                return True
+                return True    
```

Comment:
No need to change, but this PR adds a lot of whitespace at the end of the line. It surprises me since usually whitespaces at the end are removed. Could this be a setting in your editor that adds these whitespaces?

Reply:
I didn't edit this file.  It's from @Pugwash1 via the jmriusers file area.